www/api/search

```diff
@@ -41,6 +41,7 @@ operators on one type of object; this must be one of the following:
    <li><b>poll</b> - search for a poll
    <li><b>comp</b> - search for a competition/award event
    <li><b>member</b> - search the member directory
+   <li><b>tag</b> - search the list of tags
```

Comment:
Is it really a "search" API? I thought it's just a listing.

Reply:
<https://ifdb.org/search?tag&searchfor=fiction> returns tags matching your search, sorted by relevance. That's a search API if I ever heard one.